.travis.yml

```diff
@@ -24,11 +15,17 @@ matrix:
       osx_image: xcode7.3
       language: generic
       env: TOXENV=py35 BDIST=1
+      addons:
+        apt:
+          sources:
+            # Debian sid currently holds OpenSSL 1.1.0
+            # change this with future releases!
+            - debian-sid
+          packages:
+            - libssl-dev
     - python: 3.5
       env: TOXENV=py35 BDIST=1
     - python: 3.5
-      env: TOXENV=py35 NO_ALPN=1
```

Comment:
Don't delete the section that wouldn't have functioning ALPN! Otherwise we'll have the problem I reported all over again later.